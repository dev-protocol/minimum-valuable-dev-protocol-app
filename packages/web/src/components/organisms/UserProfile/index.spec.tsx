import React from 'react'
import Jdenticon from 'react-jdenticon'
import { render } from '@testing-library/react'
import { UserProfile } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-for-apps/hooks')
jest.mock('react-jdenticon')

describe(`${UserProfile.name}`, () => {
  test('Snapshot', () => {
    ;(Jdenticon as jest.Mock).mockImplementation(() => <></>)
    const component = render(<UserProfile />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
