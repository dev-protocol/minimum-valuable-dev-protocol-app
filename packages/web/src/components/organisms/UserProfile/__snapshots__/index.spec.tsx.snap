// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-1wzqvkc-0 byYDMh"
    >
      <section
        class="sc-4avdzt-0 jbRCmr"
      >
        <div
          class="sc-4avdzt-1 jGXkIY"
        >
          Your Address
        </div>
        <div
          class="sc-4avdzt-2 daxNIz"
        >
          -
        </div>
      </section>
      <section
        class="sc-4avdzt-0 jbRCmr"
      >
        <div
          class="sc-4avdzt-1 jGXkIY"
        >
          Display name
        </div>
        <div
          class="sc-4avdzt-2 daxNIz"
        >
          (Not set)
        </div>
      </section>
      <section
        class="sc-4avdzt-0 jbRCmr"
      >
        <form
          class="ant-form ant-form-horizontal sc-4avdzt-3 jVvSPm"
        >
          <div
            class="ant-row ant-form-item"
          >
            <div
              class="ant-col ant-form-item-control"
            >
              <div
                class="ant-form-item-control-input"
              >
                <div
                  class="ant-form-item-control-input-content"
                >
                  <input
                    class="ant-input"
                    id="name"
                    placeholder="Enter the new display name"
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
          <button
            class="ant-btn ant-btn-primary"
            type="submit"
          >
            <span>
              Save
            </span>
          </button>
        </form>
      </section>
    </div>
  </div>
</body>
`;
